packages/Panel/stories/examples/BasicPanel.js

```diff
@@ -22,7 +22,7 @@ export default function BasicPanel() {
           <TextLine repeat={100} />
         </Panel.Content>
         <Panel.Footer>
-          <Button>Default action</Button>
+          <Button style={{ marginRight: "8px" }}>Default action</Button>
```

Comment:
Nice! Just one thing here, I believe we can put this `margin-right: ${tokens.space}` for all buttons except the last one into Footer.styles.js
After that, we should be able to remove the inline stylings in NestedPopOverPanel.js, PushContentPanel.js, Showcase.js.